deltacat/compute/stats/basic.py

```diff
@@ -21,7 +21,7 @@
 
 def collect(
         source_partition_locator: PartitionLocator,
-        delta_stream_position_range_set: Set[Tuple[int, int]],  # TODO: R&D on RangeSet impl
+        delta_stream_position_range_set: Optional[Set[Tuple[UnboundedRange, UnboundedRange]]] = {(None, None)},
```

Comment:
```suggestion
        delta_stream_position_range_set: Optional[Set[Tuple[DeltaRange, DeltaRange]]] = {(None, None)},
```
Also, we should avoid assigning mutable collections as defaults since this can lead to some hard-to-troubleshoot bugs. One simple resolution is to just set this to `None` and assign the default in one of the first lines of the function impl.

Reply:
Good catch, thanks 🙂